### DISCUSSION TITLE
How to add a new dependency? What has happened with the UI?

### DISCUSSION THREAD

**Discussion by OnkelTem** [codesandbox/codesandbox-client#7002] [General]
Hi

I entered codesandbox today and found UI totally screwed.

I see only this:

![image](https://user-images.githubusercontent.com/114060/194756939-17814e6b-3d73-4b25-8d71-7686b3776286.png)

I.e. no any controls on adding a dependency. Seems like someone has broken the UI.
I also visited all the menus but found nothing there as well.

Any ideas?

**Comment by JamesACS** [comment]
Hey @OnkelTem 

It looks like you're accessing our new repositories interface. Are you still having scaling issues?

**Comment by michael-mcgreal** [comment]
@OnkelTem that is the new cloud interface. Look for a "+" icon around the top right, then select terminal. From the terminal you can add packages using a package manager. i.e. `yarn add lodash` or `npm i lodash`. And they will appear in your package.json file.
